tests/mocks/stubCreateEthers.ts

```diff
@@ -0,0 +1,12 @@
+import sinon from 'sinon';
+import { TokensController } from '../../src/assets/TokensController';
+
+const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
+  return sinon.stub(ctrl, '_createEthersContract').callsFake(() => {
+    return {
+      supportsInterface: sinon.stub().returns(res),
+    } as any;
```

Comment:
Wish I could do better than this, but ethers.Contract makes the contract read functions as `readonly` and that makes it really hard to return anything here that will satisfy the full interface.